### PR TITLE
Under Construction Page

### PR DESCRIPTION
**Pull request by YanZ777**
feat: Create a new under construction page. Route the schedule of events and the travel and accommodations pages to that under construction page.

## Desktop View
image

## Mobile View
image

### CODE REVIEW

File: web/src/pages/UnderConstructionPage.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import styles from 'huyan/pages/UnderConstructionPage.css';
+import BodyContainer from 'huyan/components/BodyContainer/BodyContainer';
+
+function UnderConstructionPage() {
+   return (
+      <BodyContainer>
+         <div className={styles.underConstructionContainer}>
+            <div className={styles.constructionEmoji}>🚧</div>
+            <h1>Under Construction</h1>
+            <div>Please return at a later point for updates.</div>
```
Comment (huy-vuong) [inline web/src/pages/UnderConstructionPage.jsx:12]: **suggestion:**
```suggestion
            <div>Please return later for updates.</div>
```

File: web/src/pages/UnderConstructionPage.test.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import UnderConstructionPage from 'huyan/pages/UnderConstructionPage';
+
+describe('UnderConstructionPage', () => {
+   it('renders the UnderConstruction page', async () => {
+      memoryRender(<UnderConstructionPage />);
+
+      expect(await screen.findByText('🚧')).toBeInTheDocument();
+      expect(await screen.findByText('Under Construction')).toBeInTheDocument();
+      expect(
+         await screen.findByText('Please return at a later point for updates.'),
```
Comment (huy-vuong) [inline web/src/pages/UnderConstructionPage.test.jsx:14]: **suggestion:**
```suggestion
         await screen.findByText('Please return later for updates.'),
```

File: web/src/pages/UnderConstructionPage.css [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.underConstructionContainer {
+   text-align: center;
+}
+
+.constructionEmoji {
+   font-size: 200px;
+   font-family: 'Noto emoji';
```
Comment (huy-vuong) [inline web/src/pages/UnderConstructionPage.css:7]: **nitpick:** I think it makes sense to either capitalize font names as they're shown online or not capitalize them at all. My personal preference is to capitalize them.
```suggestion
   font-family: 'Noto Emoji';
```

File: web/package.json [hunk 4]
```diff
@@ -53,6 +53,7 @@
    "dependencies": {
       "@fontsource/lexend": "^4.5.13",
       "@fontsource/playfair-display": "^4.5.12",
+      "@fontsource/noto-emoji": "5.0.3",
```
Comment (huy-vuong) [inline web/package.json:56]: **issue:** I think like for Lexend, we'll need to add an import in `global.css`.

```css
@import '@fontsource-variable/noto-emoji';
```

**Review by huy-vuong — CHANGES_REQUESTED** [review]

File: web/package.json (hunk 4 above)
Comment (YanZ777) [reply web/package.json:56]: This doesn't seem to work for me.

Comment (huy-vuong) [reply web/package.json:56]: Oops, I copy-pasted from #296 with the `-variable` bit of it. I meant to ask to include that import in `global.css`, not to necessarily request to use a variable font for Noto Emoji (though it'd be cool if we did).

Here's the import I meant to ask for:

```css
@import '@fontsource/noto-emoji';
```

**Review by huy-vuong — CHANGES_REQUESTED** [review]

File: web/src/index.jsx [hunk 5]
```diff
@@ -21,6 +24,14 @@ function Huyan() {
                <Route path="/" element={<App />} />
                <Route path="/rsvp" element={<RsvpPage />} />
                <Route path="/faq" element={<FaqPage faqs={Faqs} />} />
+               <Route
+                  path="/schedule-of-events"
+                  element={<UnderConstructionPage />}
+               />
+               <Route
+                  path="/travel-and-accommodations"
```
Comment (huy-vuong) [inline web/src/index.jsx:32]: **note:** Not worth blocking this PR over, but I wonder if we should shorten these later when we make the real things.
```suggestion
                  path="/schedule"
                  element={<UnderConstructionPage />}
               />
               <Route
                  path="/travel"
```

**Review by huy-vuong — APPROVED** [review]

**Pull request merged**